app/Http/Controllers/PeopleController.php

```diff
@@ -1540,6 +1556,38 @@ public function storeDebt(Request $request, $contactId)
         return redirect('/people/' . $contact->id);
     }
 
+    /**
+     * Update stored debt.
+     * @param  Request $request
+     * @param  int $contactId
+     * @param  int $debtId
+     */
+    public function updateDebt(Request $request, $contactId, $debtId)
+    {
+        $contact = Contact::findOrFail($contactId);
+        $debt = Debt::findOrFail($debtId);
+
+        if ($contact->account_id !== Auth::user()->account_id) {
+            return redirect()->route('people.index');
+        }
+
+        if ($debt->contact_id !== $contact->id) {
+            return redirect()->route('people.index');
+        }
+        
+        $debt->in_debt = $request->input('in-debt');
+        $debt->amount = $request->input('amount');
+        $debt->reason = $request->input('reason');
+
+        $debt->save();
+
+        $contact->logEvent('debt', $debt->id, 'edit');
```

Comment:
Indeed, this should be `update`. If it's `edit` it's gonna raise an error on the Dashboard.